libs/cardano-ledger-conformance/src/Test/Cardano/Ledger/Conformance/ExecSpecRule/Conway/Certs.hs

```diff
@@ -23,38 +23,45 @@ import qualified Data.Text as T
 import qualified Lib as Agda
 import Test.Cardano.Ledger.Conformance
 import Test.Cardano.Ledger.Conformance.ExecSpecRule.Conway.Base
+import Test.Cardano.Ledger.Conformance.ExecSpecRule.Conway.GovCert ()
 import Test.Cardano.Ledger.Constrained.Conway
+import Test.Cardano.Ledger.Constrained.Conway.WitnessUniverse
 import Test.Cardano.Ledger.Imp.Common hiding (context)
 
 instance
   IsConwayUniv fn =>
   ExecSpecRule fn "CERTS" Conway
   where
-  type ExecContext fn "CERTS" Conway = ConwayCertExecContext Conway
+  type ExecContext fn "CERTS" Conway = (WitUniv Conway, ConwayCertExecContext Conway)
```

Comment:
It'd be better to add a witness universe field to `ConwayCertExecContext` and then get rid of the pair here

Reply:
same as before.